crates/bevy_internal/Cargo.toml

```diff
@@ -77,7 +77,7 @@ serialize = [
 multi-threaded = [
   "bevy_asset/multi-threaded",
```

Comment:
```suggestion
  "bevy_asset?/multi-threaded",
```
could you also change it for bevy_asset?

Reply:
Done